Pod/Classes/Cards/Default Card Types/ChinaUnionPay.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ChinaUnionPay.swift
+//  Pods
+//
+//  Created by Shiyuan Jiang on 3/30/16.
+//
+//
+
+import UIKit
+
+public struct ChinaUnionPay: CardType {
```

Comment:
Thank you for pointing that out! I do think we need documentations but for this class since it is already clean and self-explanatory, it might be better to leave it as it is :D